### PR TITLE
add 1.34.x to compatibility matrix

### PR DESCRIPTION
**Pull request by frzifus**
Signed-off-by: Benedikt Bongartz <bongartz@klimlive.de>

### CODE REVIEW

**Review by rubenvp8510 — DISMISSED** [review]

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ The following table shows the compatibility of jaeger operator with different co
 
 | Jaeger Operator | Kubernetes           | Strimzi Operator   | Cert-Manager |
 |-----------------|----------------------|--------------------|--------------|
-| v1.33           | v1.19, v1.20, v1.21  | v0.23              | v1.6.1       |
+| v1.34.x         | v1.19 to v1.23       | v0.23              | v1.6.1       |
```
Comment (iblancasa) [inline COMPATIBILITY.md:6]: We are testing Kubernetes 1.24 too. Should we place it here?

**Review by iblancasa — CHANGES_REQUESTED** [review]

File: COMPATIBILITY.md (hunk 1 above)
Comment (frzifus) [reply COMPATIBILITY.md:6]: good catch. yes :)

[the remark this answers is not in the document]
Comment (frzifus) [reply COMPATIBILITY.md:6]: makes sense :+1:

**Comment by frzifus** [comment]
like this @rubenvp8510 @iblancasa ?

**Comment by iblancasa** [comment]
Awesome!

**Review by iblancasa — APPROVED** [review]

**Review by rubenvp8510 — APPROVED** [review]

**Comment by frzifus** [comment]
it seems that the tests are not executed, since nothing has changed in the code?

**Comment by iblancasa** [comment]
@frzifus yes. It was done to save resources

**Comment by KeithTt** [comment]
Is there a value for v1.16?

**Pull request merged**